worlds/atacama_y1a.world

```diff
@@ -17,11 +17,11 @@
     <physics type='ode'>
       <ode>
         <solver>
-          <type>quick</type>
+          <type>world</type>
         </solver>
       </ode>
-      <max_step_size>0.0025</max_step_size>
-      <real_time_update_rate>400</real_time_update_rate>
+      <max_step_size>0.00625</max_step_size>
+      <real_time_update_rate>160</real_time_update_rate>
```

Comment:
I'm concerned that this is too close to the edge of stability. Since we can't test every possible arm movement, there might be a corner case that gives us instability that we just haven't discovered yet. Should we leave some breathing room by running at 180Hz or 200Hz instead?

Reply:
I think you are right, this is the lowest I could get, and it might be unsafe for future developments. I'd go with 200Hz, such that the inverse is a finite number (1/200=0.005) and we can get a real time factor of exactly 1.

Reply:
200 Hz sounds safe to me. It will still be double the performance that we had before.

Reply:
Done!